tests/conftest.py

```diff
@@ -123,14 +123,16 @@ def crossbar_config(tmpdir, pytestconfig):
             yaml.safe_dump(conf, stream)
 
 @pytest.fixture(scope='function')
-def crossbar(tmpdir, crossbar_config):
-    if not find_spec('crossbar'):
-        pytest.skip("crossbar not found")
+def crossbar(tmpdir, pytestconfig, crossbar_config):
+    crossbar_venv = Path(pytestconfig.getoption("--crossbar-venv"))
+    if not crossbar_venv.is_absolute():
+        crossbar_venv = pytestconfig.rootdir / crossbar_venv
+    crossbar_bin = crossbar_venv / "bin/crossbar"
```

Comment:
Wouldn't it be simpler to just pass the crossbar_bin path via an option? We're not really interested in the venv otherwise.

Reply:
True, I'll change that.